src/backend/distributed/utils/citus_stat_tenants.c

```diff
@@ -586,6 +631,19 @@ CreateSharedMemoryForMultiTenantMonitor()
 						  monitor->namedLockTranche.trancheName);
 	LWLockInitialize(&monitor->lock, monitor->namedLockTranche.trancheId);
 
+	HASHCTL info;
+
+	memset(&info, 0, sizeof(info));
+	info.keysize = sizeof(TenantStatsHashKey);
+	info.entrysize = sizeof(TenantStats);
+	info.hash = TenantStatsHashFn;
+	info.match = TenantStatsMatchFn;
+
+	monitor->tenants = ShmemInitHash("citus_stats_tenants hash",
+									 StatTenantsLimit * 3, StatTenantsLimit * 3,
+									 &info, HASH_ELEM | HASH_FUNCTION | HASH_COMPARE |
```

Comment:
By using `HASH_BLOBS` we can we can remove the `hash` and `match` functions, assuming that TenantStatsHashKey passes the `assert_valid_hash_key2` check. See `hash_helpers.{c,h}`